pkg/platform/abstract/platform.go

```diff
@@ -128,8 +128,10 @@ func (ap *Platform) HandleDeployFunction(existingFunctionConfig *functionconfig.
 	// clear build mode
 	createFunctionOptions.FunctionConfig.Spec.Build.Mode = ""
 
+	skipDeploy := functionconfig.SkipDeploy(createFunctionOptions.FunctionConfig.Meta.Annotations)
```

Comment:
This should move to the top of the function, unless some special reason not to 
Together with `functionconfig.SkipBuild()` for consistency.